mint-002.md

```diff
@@ -0,0 +1,238 @@
+# mint-002
+
+## 5 Key Degrading Multisig
+
+## Motivation
+
+Expanding upon the concept of a [2-of-3 key timed multisig](mint-001.md) [(mint-001)](mint-001.md), a 5 key timed multisig allows for multiple timelocks to be introduced for additional flexibility.
+
+A timelock can be employed to allow a native 3-of-5 multisig to become 2-of-5, and eventually 1-of-5 for disaster recovery. 
+
+This can be employed by using a miniscript `tresh()` with seven conditions:
+
+### 3 Layer Conditions to be satisfied using:
+
+#### Base expressions (type B). 
+
+1. **timelock**: **after(**int**)**[^after] or **older(**int**)** - either **relative** or **absolute** [^timelock].
+
+2. **timelock**: **after(**int**)** or **older(**int**)**[^older]  - either **relative** or **absolute** [^either].
+
+#### Key expressions[^k_type] (type K).
+
+1. **pk(**key1**)** [^pk_key1]
+
+2. **pk(**key2**)** [^pk_key2]
+
+3. **pk(**key3**)** [^pk_key3]
+
+4. **pk(**key4**)** [^pk_key4]
+
+5. **pk(**key5**)** [^pk_key5]
+
+
+### More on Timelock Values
+
+- The security of the descriptor template is not dependent on the value of a relative or absolute timelock, as it only impacts the duration of the timelock.
+    
```

Comment:
In 97142059eea7312d73ee8c710b31db2a1a94318a:

Trailing whitespace here and in the following blank lines.

Reply:
resolved in: 5b7a4e5fcbae45e5e1fce0106ed799e43bea6f0d